interfaces/builtin/dsp.go

```diff
@@ -49,10 +49,16 @@ const ambarellaDspConnectedPlugApparmor = `
 # processor
 /dev/ucode rw,
 
-# The iav device node is the device node exposed for the specific IAV linux
-# device driver used on the ambarella device
+# The iav device node is the device node exposed for the specific IAV driver
+# that includes a set of Linux device drivers that encapsulate the lower-level
+# complexities of the CV2x / S6Lm DSP core functionalities
 /dev/iav rw,
 
+# The cavalry device node is the driver for the CV2x vector processor (VP)
+# that includes a set of Linux device drivers that encapsulate the lower-level
+# complexities of the CV2x's VP functionalities
```

Comment:
To @pedronis' point, this comment is a bit confusing. I'm not sure if `/dev/cavalry` is itself a device node used to access low-level complexities and such on the vector processor or if it is actually a higher level thing and abstracts away the low-level complexities. In either case, it doesn't really matter for this file, I think we should just use something like this:

```suggestion
# The cavalry device node is used for managing the CV2x vector processor (VP).
```